doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating stream-based
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
```

Comment:
@joyeecheung In that case we could just refer to `createConnection` :smile:

But, yeah, this is getting really bike-shed-y and I trust your judgement on this :)